plugins/generator-1.18.2/forge-1.18.2/procedures/entity_islooking_at_entity.java.ftl

```diff
@@ -0,0 +1,16 @@
+(new Object() {
+	public boolean isLooking(Entity _source, Entity _target, double area, boolean ignoreBlocks) {
+		Vec3 vec3 = _source.getViewVector(1.0F).normalize();
+		Vec3 vec31 = new Vec3(_target.getX() - _source.getX(), (_target.getEyeY() - _target.getEyeHeight() * 0.5) - _source.getEyeY(),
+				_target.getZ() - _source.getZ());
+		double d0 = vec31.length();
+		vec31 = vec31.normalize();
+		double d1 = vec3.dot(vec31);
+		if (_source instanceof LivingEntity _liv) {
+		if (ignoreBlocks)

```

Comment:
There is a missing indentation here (and so the line after too) and on the other file too.